### PR TITLE
Register page validations

### PR DESCRIPTION
**Pull request by NPBogdan**
Validations on the register page, some classes for appearance and a new "Form request" class.

### CODE REVIEW

File: app/Http/Controllers/RegisterController.php [hunk 1]
```diff
@@ -12,14 +13,19 @@ public function showForm(){
...
                 'name' => $request->input('name'),
+                'first_name' => $request->input('firstName'),
                 'email' => $request->input('email'),
-                'password' => $password
+                'password' => $password,
+                'address' => $request->input('address'),
+                'birthday' => $request->input('birthday'),
                 ]);
+
+        echo "A mers";
```
Comment (andreisamoila74) [inline app/Http/Controllers/RegisterController.php]: why is this here?

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/RegisterController.php (hunk 1 above)
Comment (NPBogdan) [reply app/Http/Controllers/RegisterController.php]: I fix that
